styles.css

```diff
@@ -0,0 +1,1240 @@
+
+}
+
+#tweak-head4 {
+    grid-area: head;
+    display: inline;
+    background-color: black;
+    margin-bottom: 890%;
+    margin-top: 10%;
+    margin-left: 14%;
+    margin-right: 10%;
+    border-radius: 60%;
+}
+
+#tweak-head4 img {
+    object-fit: contain;
+    height: 100%;
+    width: 100%;
+    border-radius: 60%;
+}
+
+#tweak-wings4 {
+    grid-area: header;
+    padding-top: 2mm;
+}
+
+#tweak-body4 {
+    grid-area: body;
+    padding-top: 0.5mm;
+}
+
+#tweak-body4-pic {
+    grid-area: pic;
+    margin-top: 2mm;
+    border-radius: 25px;
+
+}
+
+#tweak-body4-pic img {
+    width: 97%;
+    height: 500px;
+    object-fit: fill;
+    border-radius: 15px;
+}
+
+#tweak-tail4 {
+    grid-area: dots;
+    padding-top: 2mm;
+}
+
+#tweak-branch4 {
+    grid-area: branch;
+    display: grid;
+    grid-template-columns: repeat(7, 1fr);
+    max-width: 490px;
+}
+
+#tweak-branch4 p {
+    font-weight: bold;
+    font-size: 3mm;
+    font-family: Arial, Helvetica, sans-serif;
+    color: #a2a2a2;
+
+}
+
+#reply4,
+#number-reply4,
+#retweak4,
+#number-retweaks4,
+#like4,
+#number-likes4,
+#share4 {
+    width: 23px;
+    margin-top: 3mm;
+
+}
+
+#number-reply4,
+#number-retweaks4,
+#number-likes4 {
+    margin-top: 4mm;
+    justify-items: left;
+    margin-right: 2.6cm;
+
+}
+
+
+
+.tweak5 {
+    border-left: 2px solid rgb(204 215 221);
+    border-right: 2px solid rgb(204 215 221);
+    display: grid;
+    grid-template: "head header dots"30px "head  body  body"auto "head  pic    pic"auto "head   branch  branch"50px/ 10% 1fr 6%;
+
+}
+
+#tweak-head5 {
+    grid-area: head;
+    display: inline;
+    background-color: #a2a2a2;
+    margin-bottom: 980%;
+    margin-top: 10%;
+    margin-left: 14%;
+    margin-right: 10%;
+    border-radius: 60%;
+}
+
+#tweak-head5 img {
+    object-fit: cover;
+    height: 100%;
+    width: 100%;
+    border-radius: 60%;
+}
+
+#tweak-wings5 {
+    grid-area: header;
+    padding-top: 2mm;
+}
+
+#tweak-body5 {
+    grid-area: body;
+    padding-top: 0.5mm;
+}
+
+#tweak-body5-pic {
+    margin-top: 2mm;
+    grid-area: pic;
+}
+
+#tweak-body5-pic img {
+    width: 97%;
+    height: 600px;
+    border-radius: 5%;
+}
+
+#tweak-tail5 {
+    grid-area: dots;
+    padding-top: 2mm;
+}
+
+#tweak-branch5 {
+    grid-area: branch;
+    display: grid;
+    grid-template-columns: repeat(7, 1fr);
+    max-width: 490px;
+}
+
+#tweak-branch5 p {
+    font-weight: bold;
+    font-size: 3mm;
+    font-family: Arial, Helvetica, sans-serif;
+    color: #a2a2a2;
+
+}
+
+#reply5,
+#number-reply5,
+#retweak5,
+#number-retweaks5,
+#like5,
+#number-likes5,
+#share5 {
+    width: 23px;
+    margin-top: 3mm;
+
+}
+
+#number-reply5,
+#number-retweaks5,
+#number-likes5 {
+    margin-top: 4mm;
+    justify-items: left;
+    margin-right: 2.6cm;
+
+}
+
+#tweak-body1 span:hover {
+    text-decoration: underline;
+}
+
+#tweak-body4 span:hover {
+    text-decoration: underline;
```

Comment:
This is another example of repeated styles. You could create one class called `hover-underline` or something similar, add the text decoration when it's hovered: 
```css
.hover-underline:hover {
  text-decoration: underline;
}
``` 

Then apply it to each of the elements that you want to be underlined when they're hovered. That way you can achieve the same result with only one class.